ipaserver/plugins/realmdomains.py

```diff
@@ -36,6 +36,16 @@
 
 Manage the list of domains associated with IPA realm.
 
+This list is useful for Domain Controllers from other realms which have
+established trust with this IPA realm. They need the information to know
+which request should be forwarded to KDC of this IPA realm.
+
+Automatic management: a domain is added to realm domains list automatically
+when new DNS Zone managed by IPA is created. Same applies for deletion.
```

Comment:
nitpicking, but I would rephrase:
a domain is automatically added to the realm domains list when a new DNS Zone managed by IPA is created. Same applies for deletion.